zebra/rt_netlink.c

```diff
@@ -675,9 +684,13 @@ int netlink_route_change(struct sockaddr_nl *snl, struct nlmsghdr *h,
 			 ns_id_t ns_id, int startup)
 {
 	int len;
-	vrf_id_t vrf_id = ns_id;
```

Comment:
This is a bit of a weird interaction, get a `vrf_id` to only use for debug, why not just change the debug to say this is the namespace id and skip this weird lookup we have here?

Reply:
I wil do it.